lib/css/CssParser.js

```diff
@@ -304,8 +336,11 @@ class CssParser extends Parser {
 			isSelector: () => {
 				return mode !== CSS_MODE_IN_RULE && mode !== CSS_MODE_IN_LOCAL_RULE;
 			},
-			url: (input, start, end, contentStart, contentEnd) => {
-				const value = cssUnescape(input.slice(contentStart, contentEnd));
+			url: (input, start, end, contentStart, contentEnd, isString) => {
```

Comment:
Can we typedef these functions? It's hard to tell from the PR if types come from/or are provided by `walkCssTokens`

Reply:
@TheLarkInn We already have, look at `CssTokenCallbacks` type